tests/integration/test_cli_benchmark_analytics_undo_display_utils.py

```
@@ -0,0 +1,2619 @@
+        )
+
+        def fake_runner(files: list[Path]) -> _SuiteIterationOutcome:
+            return _SuiteIterationOutcome(processed_count=2)
+
+        def fake_classifier(
+            files: list[Path], outcome: _SuiteIterationOutcome
+        ) -> _SuiteExecutionClassification:
+            return _SuiteExecutionClassification(effective_suite="io", degraded=False)
+
+        mock_console = MagicMock()
+        elapsed, count, classification = _execute_suite_iteration(
```

Ruff (0.15.9)

[warning] Unpacked variable `elapsed` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)